Mathlib/Data/Finset/Sups.lean

```diff
@@ -178,6 +188,21 @@ theorem subset_sups {s t : Set α} :
   subset_image₂
 #align finset.subset_sups Finset.subset_sups
 
+lemma image_sups (f : F) (s t : Finset α) : image f (s ⊻ t) = image f s ⊻ image f t :=
+  image_image₂_distrib $ map_sup f
+
+lemma map_sups (f : F) (hf) (s t : Finset α) :
+    map ⟨f, hf⟩ (s ⊻ t) = map ⟨f, hf⟩ s ⊻ map ⟨f, hf⟩ t := by
+  simpa [map_eq_image] using image_sups f s t
```

Comment:
I think the awkward `map ⟨f, hf⟩` phrasing combined with the fact this assumes `[DecidableEq β]` means this lemma is probably never going to be useful.

Reply:
At the point you're carrying around a separate `hf` assumption though, you can use `card_image` instead.

To be clear, I don't think this lemma does any harm, but I am surprised it helps you.